vars/startEnvironmentIfRequired.groovy

```diff
@@ -0,0 +1,40 @@
+import uk.gov.hmcts.contino.GithubAPI
+import uk.gov.hmcts.contino.HealthChecker
+import groovy.json.JsonSlurper
+
+def call(Map params) {
+  def environment = params.environment
+  def subscription = params.subscription  
+  def subscriptionName = params.aksSubscription.name
+  def businessArea = env.BUSINESS_AREA_TAG
+  def clusterResourceGroup = env.AKS_RESOURCE_GROUP
+  def clusterName = env.AKS_CLUSTER_NAME
+  
+  if(subscriptionName.toLowerCase().contains("sbox")){
+    log.info("Checking AKS Environment: $environment, Subscription is $subscriptionName, in business area: $businessArea")
+    def azCommand = { cmd -> return sh(script: "env AZURE_CONFIG_DIR=/opt/jenkins/.azure-jenkins az $cmd", returnStdout: true).trim() }
+    azCommand 'login --identity > /dev/null'
```

Comment:
pretty sure you don't need this at this point? it will have been done earlier, this can take quite awhile.

I think there's some code somewhere that can tell you if it's already logged in too and only do it then if needed.

Reply:
True - https://github.com/hmcts/cnp-jenkins-library/pull/1177/files/03e1f01d6e61ffc0296a33b915fb9af2eabeb03a#diff-debf4f95cedccc768d522a59c5a9b2a7e40f2ab05cda7a5d430647976f82ea26R16 withSubscription is already used so just changing subscription instead